content/publish/consumerParameters.json

```diff
@@ -0,0 +1,65 @@
+{
+  "consumerParameters": {
+    "fields": [
+      {
+        "name": "consumerParameters",
+        "label": "Custom parameters",
+        "type": "consumerParameters",
+        "required": false,
+        "fields": [
+          {
+            "name": "name",
+            "label": "Parameter Name",
+            "placeholder": "custom_parameter",
```

Comment:
let's give this placeholders a better description. To this and "label" please.

Reply:
Yeah I agree lets use better place holders. 

On another matter, why do we need the user to enter both a `name` and a `label`? We can just use the label as the name. If you just want to avoid spaces in the name you can use `.replace(/ /g,"_");`